style.css

```diff
@@ -0,0 +1,235 @@
+  justify-content: end;
+}
+
+nav {
+  align-items: center;
+  position: fixed;
+  top: 0%;
+  width: 100%;
+  height: 50px;
+  background-color: #ffffff;
+  justify-content: space-between;
+  z-index: 1;
+}
+
+#left-navbar {
+  justify-content: space-evenly;
+  width: 12%;
+  align-items: center;
+  margin-left: 15px;
+}
+
+#mid-navbar {
+  align-self: center;
+  margin-right: 20px;
+  width: 50%;
+  justify-content: center;
+}
+
+#search-bar {
+  background-color: white;
+  width: 85%;
+  padding: 5px 10px;
+  border: 1px solid silver;
+  border-radius: 3px 0px 0px 3px;
+  font-size: medium;
+}
+
+#search-button {
+  border: 1px solid gray;
+  padding: 10px 30px;
+  border-radius: 0px 3px 3px 0px;
+}
+
+#right-navbar {
+  width: 150px;
+  justify-content: space-between;
+  margin-right: 40px;
+}
+
+.main-video {
+  margin-left: initial;
+}
+
+.flex {
+  display: flex;
+}
+
+.flex-column {
+  flex-direction: column;
+}
+
+.space-between {
+  justify-content: space-between;
+}
+
+.flex-horizontal {
+  flex-direction: row;
+}
+
+.upcoming {
+  font-family: Roboto, Arial, “Sans Serif”, Noto, “YT Sans”, “Roboto Mono”;
+  font-size: 16px;
+  color: #030303;
+}
+.input {
+  width: 550px;
+  position: relative;
+  left: 219px;
+}
+
+.container {
+  position: relative;
+  top: 85px;
+  left: 22px;
+}
+.main-fc {
+  width: 60%;
+}
+
+.other-videos {
+  width: 30%;
+}
+
+#autobutton {
+  border: none;
+  background-color: transparent;
+}
+.sidebar {
+  position: relative;
+  right: 35px;
+  top: 80px;
+}
+.image {
+  float: left;
+  height: 10px;
+  position: relative;
+}
+.aside-video {
+  grid-template-columns: 180px 180px;
+  display: grid;
+  margin-bottom: 4px;
+}
+.aside-description {
+  font-family: Roboto;
+  line-height: 1.2em;
+  font-size: 12px;
+}
+.song-title {
+  font-family: Roboto;
+  line-height: 1.2em;
+  font-size: 14px;
+  color: #030303;
+  display: inline-block;
+}
+.aside-description {
+  position: relative;
+  bottom: 8px;
+}
+.artist-name {
+  font-family: Roboto;
+  line-height: 1.2em;
+  font-size: 13px;
+  color: #606060;
+}
+.aside-views {
+  font-family: Roboto;
+  line-height: 1.2em;
+  font-size: 13px;
+  color: #606060;
+}
+.video-duration {
+  position: relative;
+  float: right;
+  bottom: 18px;
+  right: 14px;
+  font-size: 12px;
+  background-color: black;
+  color: whitesmoke;
+}
+
+.images {
+  border-radius: 50%;
+}
+
+.smaller-gray {
+  font-size: 0.9rem;
+  color: gray;
+}
+
+.flex-horizontal {
+  flex-direction: row;
+}
+
+#video-below {
+  justify-content: space-between;
+  align-items: center;
+}
+
+.video-section {
+  justify-self: start;
+}
+
+.subscribe-button {
+  float: right;
+  margin-top: -120px;
+  background-color: #cc0000;
+  font-size: smaller;
+  border: none;
+  color: white;
+  padding: 10px;
+}
+
+.align-center {
+  align-items: center;
+  align-content: center;
+  align-self: center;
+}
+
+#autoplay {
+  display: inline;
+}
+
+.inline {
+  display: inline;
+}
```

Comment:
this is duplicate you can use only the class if you want please remove the id selector